kmeans1.py

```diff
@@ -5,10 +5,9 @@
 
 Input : A dataset of 100 randomly generated datapoints
 Output : 'k' number of datapoints representing the centroids of the clusters
-
 '''
 
-import random, math, time, numpy, os
+import random, math, os
 os.system('clear')
 
 
```

Comment:
I cannot comment on individual lines, hence commenting here :

1. Very good naming conventions, except for few places, like : meanx ( where initially it is not yet mean, it could have been something like sumX )

2. Try to externalize the constants to any kind of file or anything else. Don't hard code the values in code.

3. I think you missed to re-initialize the 'cluster' variable in the loop. It is currently adding the points to an existing cluster. It will give you wrong calculation for the centriods.

4. Name the variables differently. Example, here : the list of cluster points are named `cluster` and also the loop variable at the end is also named `cluster`.

5. Please make it modularized, i.e., define functions for everything. 

6. For every function, I would prefer comments like this :
```
 def findDistance(x_cent, y_cent, x, y):	 def findDistance(x_cent, y_cent, x, y):
        '''
           Function to find the distance between 2 points (x, y) and (x_cent, y_cent)
          (x, y) is a point in the dataset	 #(x, y) is a point in the dataset
          (x_cent, y_cent) is the centroid	 #(x_cent, y_cent) is the centroid
          @return : Distance between point (x,y) and the centroid (x_cent, y_cent )
         '''
         return int(math.sqrt((x-x_cent)*(x-x_cent) + (y-y_cent)*(y-y_cent)))
 
```